installers/rpm-code/dev-env.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/bash
 # intended to be sourced
+cd "$(dirname "${BASH_SOURCE[0]}")"
+_root() {
+    local r=$PWD
+    while [[ $(basename $r) != src ]]; do
+        d=$(dirname "$r")
+        echo $d
+        if [[ $r == $d ]]; then
+            echo 'ERROR: could not find root: using ~/src' 1>&2
+            echo ~/src
+            return 1
+        fi
+        r=$d
+    done
+    echo $r
+}
 export dev_port=2916
 export install_server=http://$(hostname -f):$dev_port
 export fedora_version=32
-export repo_rel_dir=fedora/$fedora_version/$(arch)/dev
-export rpm_code_install_dir=$HOME/src/yum/$repo_rel_dir
-export radiasoft_repo_file=$(dirname $(dirname $rpm_code_install_dir))/radiasoft.repo
+export repo_fedora_dir=$(_root)/src/yum/fedora
```

Comment:
I think this src/ isn't needed

Reply:
5fedd36

Also fixed an error in rpm_perl_install_dir.

Thanks for testing and reviewing!